portals/publisher/src/main/java/org/wso2/carbon/apimgt/ui/publisher/Util.java

```diff
@@ -0,0 +1,183 @@
+/*
+
+
+ *  Copyright (c) 2017, WSO2 LLC (http://www.wso2.org) All Rights Reserved.
```

Comment:
@DinithHerath there is formatting issue in the license header. Shall we fix this and the year as 2017 - 2023